x-pack/filebeat/fbreceiver/receiver_test.go

```diff
@@ -243,10 +244,23 @@ func TestMultipleReceivers(t *testing.T) {
 			// instance and does not interfere with others. Previously, the
 			// logger in Beats was global, causing logger fields to be
 			// overwritten when multiple receivers started in the same process.
-			r1StartLogs := zapLogs.FilterMessageSnippet("Beat ID").FilterField(zap.String("otelcol.component.id", "r1"))
+			r1StartLogs := zapLogs.FilterMessageSnippet("Beat ID").FilterField(zap.String("otelcol.component.id", "filebeatreceiver/r1"))
 			assert.Equal(c, 1, r1StartLogs.Len(), "r1 should have a single start log")
-			r2StartLogs := zapLogs.FilterMessageSnippet("Beat ID").FilterField(zap.String("otelcol.component.id", "r2"))
+			r2StartLogs := zapLogs.FilterMessageSnippet("Beat ID").FilterField(zap.String("otelcol.component.id", "filebeatreceiver/r2"))
 			assert.Equal(c, 1, r2StartLogs.Len(), "r2 should have a single start log")
+
+			meta1Path := filepath.Join(dir1, "/data/meta.json")
+			assert.FileExists(c, meta1Path, "dir1/data/meta.json should exist")
+			meta1Data, err := os.ReadFile(meta1Path)
+			assert.NoError(c, err)
+
+			meta2Path := filepath.Join(dir2, "/data/meta.json")
+			assert.FileExists(c, meta2Path, "dir2/data/meta.json should exist")
+			meta2Data, err := os.ReadFile(meta2Path)
+			assert.NoError(c, err)
+
+			assert.NotEqual(c, meta1Data, meta2Data, "meta data files should be different")
```

Comment:
I'm wondering if we should go a step further and also ensure the stores are different. By what I got from the code, the part that creates `meta.json` is different than the one that opens the store.


If we setup different IDs for each Filestream input of each Filebeatreceiver, we can open `data/registry/filebeat/log.json` and easily identify which input/store is running in each path.